satellite/console/service.go

```diff
@@ -1049,9 +1049,10 @@ func (s *Service) DeleteAPIKeys(ctx context.Context, ids []uuid.UUID) (err error
 	return nil
 }
 
-// GetAPIKeysInfoByProjectID retrieves all api keys for a given project
-func (s *Service) GetAPIKeysInfoByProjectID(ctx context.Context, projectID uuid.UUID) (info []APIKeyInfo, err error) {
+// GetAPIKeys returns paged api key list for given Project
+func (s *Service) GetAPIKeys(ctx context.Context, projectID uuid.UUID, cursor APIKeyCursor) (akp *APIKeyPage, err error) {
```

Comment:
check for invalid cursor orders

Reply:
Can you make it more clear?

Reply:
What I meant is that what if somebody specifies `cursor.OrderDirection = -2`. It seems that `sanitizeOrderDirectionName` defaults to "ASC", but I think the right approach probably is to give an error to the caller.

Although now thinking of it, I might have missed the check - maybe graphql does it already. Alternatively, maybe it's possible to specify what are the allowed values for graphql.

Note this can be done in a separate PR, since it seems there are other fields that might have similar issues.

Reply:
I think that we can move it to separate PR